src/project.rs

```diff
@@ -221,10 +221,10 @@ impl PythonProject {
         let locations = self.find_version_locations(&content)?;
 
         let prefix = if dry_run { "Would update" } else { "Updated" };
-        let mut diff = format!("{} pyproject.toml:", prefix);
+        let mut diff = format!("{prefix} pyproject.toml:");
 
         for (location, old_version) in &locations {
-            diff.push_str(&format!("\n  {}: {} → {}", location, old_version, version));
+            diff.push_str(&format!("\n  {location}: {old_version} → {old_version}"));
```

Comment:
Change to `diff.push_str(&format!("\n  {location}: {old_version} → {version}"));`